src/server.js

```diff
@@ -0,0 +1,6 @@
+const express = require("express");
```

Comment:
Can you use import instead?

Reply:
I modified it and created another pull request #38 
Could you check the modificatino at the new pull request?